external/m2/Disable-M2MaintenanceMode.ps1

```diff
@@ -0,0 +1,44 @@
+$ErrorActionPreference = "Stop"
+
+Import-Module Az.Storage
+
+function Write-Output { param ([string] $Message) $Message }
+
+function Disable-M2MaintenanceMode {
+    
+    param (
+        [parameter(Mandatory = $true, Position = 0)]
+        [ValidateSet('dev', 'test', 'prod')]
+        [string] $Environment
+    )
+
+    $resourceGroupName = "b2b-ec-$Environment"
+    $storageAccountName = "unib2becop$Environment"
```

Comment:
can we expose these names as a parameter names/prefixes instead? The runbook operator can then pass the correct values when they start a runbook job.